File: binance4j-blvt/src/main/java/com/binance4j/blvt/dto/LimitInfo.java
package com.binance4j.blvt.dto;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;

/**
 * BLVT User Limit Info.
 * 
 * @param tokenName                   Token name.
 * @param userDailyTotalPurchaseLimit User daily total purchase limit in USDT.
 * @param userDailyTotalRedeemLimit   User daily total redeem limit in USDT.
 */
@ApiModel("BLVT User Limit Info.")
public record LimitInfo(@ApiModelProperty("Token name.") String tokenName,
		@ApiModelProperty("User daily total purchase limit in USDT.") String userDailyTotalPurchaseLimit,
		@ApiModelProperty("User daily total redeem limit in USDT.") String userDailyTotalRedeemLimit) {
}

File: binance4j-blvt/src/main/kotlin/com/binance4j/blvt/dto/LimitInfo.kt
package com.binance4j.blvt.dto

import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty

/**
 * BLVT User Limit Info.
 *
 * @property tokenName                   Token name.
 * @property userDailyTotalPurchaseLimit User daily total purchase limit in USDT.
 * @property userDailyTotalRedeemLimit   User daily total redeem limit in USDT.
 */
@ApiModel("BLVT User Limit Info.")
data class LimitInfo(
    @ApiModelProperty("Token name.")
    var tokenName: String? = null,
    @ApiModelProperty("User daily total purchase limit in USDT.")
    var userDailyTotalPurchaseLimit: String? = null,
    @ApiModelProperty("User daily total redeem limit in USDT.")
    var userDailyTotalRedeemLimit: String? = null
)
